packages/browser-agent-core/src/features/jserrors/aggregate/index.js

```diff
@@ -153,16 +153,18 @@ export class Aggregate extends FeatureBase {
     var canonicalStack = this.buildCanonicalStackString(stackInfo)
 
     var params = {
-      stackHash: stringHashCode(`${stackInfo.name}_${stackInfo.message}_${canonicalStack}`),
+      stackHash: stringHashCode(canonicalStack),
       exceptionClass: stackInfo.name,
       request_uri: self.location.pathname
     }
     if (stackInfo.message) {
       params.message = '' + stackInfo.message
     }
 
-    if (!this.stackReported[params.stackHash]) {
-      this.stackReported[params.stackHash] = true
+    var bucketHash = stringHashCode(`${stackInfo.name}_${stackInfo.message}_${stackInfo.stackString}`)
```

Comment:
Would you mind adding a comment to this part of the method that explains why we use a bucket hash here, but the stack hash still gets passed on downstream via the params obj?